docker/startup

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+RCLONE_CONFIG=/config/rclone/rclone.conf
+
+if [ -z "$REMOTE_NAME" ]; then
+    echo "REMOTE_NAME is not set"
+    exit 1
+fi
+if [ -z "$REMOTE_URL" ]; then
+    echo "REMOTE_URL is not set"
+    exit 1
+fi
+if [ -z "$REMOTE_VENDOR" ]; then
+    echo "REMOTE_VENDOR is not set"
+    exit 1
+fi
+
+if [ "$REMOTE_NEXTCLOUD_CHUNK_SIZE" != "0" ]; then
+    REMOTE_NEXTCLOUD_CHUNK_SIZE="${REMOTE_NEXTCLOUD_CHUNK_SIZE}"i
+fi
+
+mkdir -p "$(dirname "$RCLONE_CONFIG")"
+
+cat <<EOF >"$RCLONE_CONFIG"
+[$REMOTE_NAME]
+type = webdav
+url = $REMOTE_URL
+vendor = $REMOTE_VENDOR
+nextcloud_chunk_size = ${REMOTE_NEXTCLOUD_CHUNK_SIZE}
```

Comment:
because this proxy is supposed to use `remote.php/webdav` endpoint we maybe should set this to 0 and not make it configurable. Chunked upload for NC is not supported on that endpoint see https://docs.nextcloud.com/server/latest/developer_manual/client_apis/WebDAV/chunking.html
and rclone seems not have implemented chunking for OC anyway